controllers/agentaction_controller.go

```diff
@@ -398,6 +398,11 @@ func (r *AgentActionReconciler) createAgentJob(ctx context.Context, log logr.Log
 	labels := r.getAgentJobLabels(action)
 	env, envFrom := r.getAgentEnv(action, agentCfg, pvc)
 	volumes, volumeMounts := r.getAgentVolumes(ctx, log, action, agentCfg, pvc, configSecret, workdirSecret, imgPullSecret)
+	var backoffLimit *int32
+	backoffLimitValue, exist := agentCfg.GetRetryLimit()
+	if exist {
+		backoffLimit = pointer.Int32Ptr(int32(backoffLimitValue))
```

Comment:
Can we define RetryLimit as int32 on the AgentConfig so that we don't need to do this casting?